app/components/appMainModule/committeePageModule/committeePage.html

```diff
@@ -1,5 +1,5 @@
 
-<div class="fluid-container">
+<div class="container">
```

Comment:
nm, I'm thinking of a conversation I had with @elinaru about the container on the homepage. The mock shows the contributor list wider than the footer, so let's change this to `container-fluid`.